docs/configuration/inc/add-enrollmentflow.md

```diff
@@ -0,0 +1,9 @@
+1. Cal-ITP creates new `EnrollmentFlow`s in the Admin:
+   - For each type of discount the new transit provider will be using, add an enrollment flow with the following:
+     - System name
+     - Label (retype the system name)
```

Comment:
It's not entirely correct to say "retype the system name", right? Like for a system name of `medicare`, we would want a label of `Medicare Cardholder`. The label is used as the radio button label in in-person enrollment, so it's a user-facing string.

I don't need to block this PR for this, but just wanted to check our understanding.

Reply:
Yeah, John and I noticed this a couple days ago when trying out this doc with RABA. I'll put off updating it at this moment so it doesn't reset the PR approval, but let's come back to it with the other updates you're raising.